package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "klaviyo-react-native-sdk",
-  "version": "1.2.0-alpha.1",
+  "version": "2.0.0-alpha.1",
```

Comment:
Up for discussion: changing our API's signature is technically breaking change. But it never woulda worked before as returning string or null...